third_party/2and3/pynamodb/attributes.pyi

```diff
@@ -20,6 +20,19 @@ class Attribute(Generic[_T]):
     def serialize(self, value: Any) -> Any: ...
     def deserialize(self, value: Any) -> Any: ...
     def get_value(self, value: Any) -> Any: ...
+    def between(self, lower: Any, upper: Any) -> Any: ...
+    def is_in(self, *values: Any) -> Any: ...
+    def exists(self) -> Any: ...
```

Comment:
BETWEEN & IN keywords are pretty board in terms of what they take (https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Condition.html) so that is why I am using Any